utils/ecosystem.js

```diff
@@ -0,0 +1,329 @@
+export default {
+  customers: [
+    {
+      name: 'Lync NFT',
+      website: 'https://www.lync.world/',
+      tags: ['NFT'],
+      announcement:
+        'https://arcana.network/blog/onboarding-lync-as-mainnet-partner/',
+      description:
+        'Lync is a gamefi infrastructure, where they are creating a one stop solution for game developers to build their games on web3 and make their NFT interoperable.',
+      image: 'images/ecosystem/customers/Lync NFT.png',
```

Comment:
remove spaces in file name

Reply:
and use small cases for consistency

Reply:
Resolved at https://github.com/arcana-network/website/pull/142/commits/b7f901ffef6eeeae575acc310f83e3e997e82cec